<ui version="4.0" >
 <class>IsePanel</class>
 <widget class="QMainWindow" name="IsePanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>353</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>ISE Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QFrame" name="frame_4" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>4</width>
        <height>4</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTabWidget" name="tabWidget" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab" >
          <attribute name="title" >
           <string>Select Board</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="1" >
            <widget class="QFrame" name="frame" >
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QLabel" name="label" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Select the ISE/SSE/JSE device to manipulate.</string>
                </property>
                <property name="wordWrap" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="board_select" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4" >
          <attribute name="title" >
           <string>ISE Console</string>
          </attribute>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QFrame" name="frame_3" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape" >
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" >
              <item row="0" column="0" colspan="4" >
               <widget class="QTextEdit" name="console_text" >
                <property name="font" >
                 <font>
                  <family>DEC Terminal</family>
                 </font>
                </property>
                <property name="readOnly" >
                 <bool>true</bool>
                </property>
                <property name="acceptRichText" >
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QPushButton" name="console_refresh" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>Refresh</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="QPushButton" name="diag0" >
                <property name="text" >
                 <string>diag0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2" >
               <widget class="QPushButton" name="diag1" >
                <property name="sizePolicy" >
                 <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text" >
                 <string>diag1</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3" >
               <spacer>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" >
                 <size>
                  <width>181</width>
                  <height>26</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_5" >
          <attribute name="title" >
           <string>Hot Swap</string>
          </attribute>
          <widget class="QFrame" name="frame_2" >
           <property name="geometry" >
            <rect>
             <x>100</x>
             <y>30</y>
             <width>212</width>
             <height>181</height>
            </rect>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Check the button to disengage the board for power off.</string>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="power_switch" >
              <property name="text" >
               <string>Power off</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>While the button is checked, the board can be powered off and replaced.</string>
              </property>
              <property name="wordWrap" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
